icu4c/source/io/unicode/ustdio.h

```diff
@@ -21,74 +21,8 @@
 ******************************************************************************
 */
 
-#ifndef USTDIO_H
-#define USTDIO_H
-
-#include <stdio.h>
-#include <stdarg.h>
-
-#include "unicode/utypes.h"
-#include "unicode/ucnv.h"
-#include "unicode/utrans.h"
-#include "unicode/localpointer.h"
-#include "unicode/unum.h"
-
-#if !UCONFIG_NO_CONVERSION
-
-/*
-    TODO
```

Comment:
I think fixing/normalizing the include guards (`__U_USTDIO_H__`) is a good idea.
However, I'm less sure about changes like this one, which shuffle around large comment blocks as it tends to "break" the first result from `git blame`.

Reply:
I believe that once the file structure is consistent throughout all files, it is easier to manage patches on top. Yes, after files lose consistency re-organization/re-factoring is required, which makes git blame unhappy, however, reaching consistency will allow easier maintenance.